core/put/source/dev_ui.cpp

```diff
@@ -305,7 +305,7 @@ namespace put
 
         void render(ImDrawData* draw_data);
 
-        bool init()
+        bool init(ImGuiStyle &style)
```

Comment:
`ImGuiStyle& style` is the convention of pmtech. please ensure consistency throughout.